contrib/mkimage-yum.sh

```diff
@@ -16,6 +16,8 @@ OPTIONS:
                    The default is "Core".
   -y <yumconf>     The path to the yum config to install packages from. The
                    default is /etc/yum.conf for Centos/RHEL and /etc/dnf/dnf.conf for Fedora
+  -f <filename>    The filename of the output tarfile
+  -c               If specified, the tarfile will be bzip2 compressed
```

Comment:
Why hard-coded for `bzip2`?

(see the approach in `mkimage.sh` for an alternate implementation https://github.com/docker/docker/blob/57977fb1273337db0e5da46faf4068c4073aa0bb/contrib/mkimage.sh#L25-L46)